gooddata-sdk/tests/catalog/test_catalog_data_source.py

```diff
@@ -284,8 +290,112 @@ def test_catalog_data_source_table(test_config):
 @gd_vcr.use_cassette(str(_fixtures_dir / "declarative_data_sources.json"))
 def test_catalog_declarative_data_sources(test_config):
     sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
-    data_sources = sdk.catalog_data_source.get_declarative_data_sources().data_sources
+    client = GoodDataApiClient(host=test_config["host"], token=test_config["token"])
+    layout_api = metadata_apis.LayoutApi(client.metadata_client)
+
+    data_sources_o = sdk.catalog_data_source.get_declarative_data_sources()
+    data_sources = data_sources_o.data_sources
 
     assert len(data_sources) == 1
     assert data_sources[0].id == test_config["data_source"]
     assert len(data_sources[0].pdm.tables) == 5
+    assert data_sources_o.to_api().to_dict() == layout_api.get_data_sources_layout().to_dict()
+
+
+@gd_vcr.use_cassette(str(_fixtures_dir / "demo_store_declarative_data_sources.json"))
+def test_store_declarative_data_sources(test_config):
+    sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
+    path = _current_dir / "store" / "declarative_data_sources"
```

Comment:
Who cleans this directory? Or should it be permanent? Is it in .gitignore then?

Reply:
It's part of .gitignore :)